python/pyproject.toml

```diff
@@ -38,6 +38,9 @@ classifiers = [
     "Topic :: Scientific/Engineering",
 ]
 
+[project.scripts]
+lance-tools="lance.cli:lance_tools_cli"
```

Comment:
I'm not sure I like the idea of having lance_tools bundled directly in `pylance`. Maybe it should just be a separate binary instead?

Reply:
Hi @Xuanwo , thanks your comments! I think there are 2 main reasons why we need bundle lance-tools in pylance:

1. Ease of Use: The simplest and most common way to use Lance is by running ```pip install pylance```. Now, after installing pylance, we can directly use the lance-tools command without needing any additional installations.
2. Version Consistency: If lance-tools and pylance are installed separately, users need to ensure that the version of lance-tools is compatible with pylance.